### PR TITLE
added a webpage that talks about python lists

### PR DESCRIPTION
**Pull request by Beatrix-droid**
added the desired webpage

### CODE REVIEW

**Comment by ishaan010** [comment]
Oh! My bad! I forgot to tell you about [this](https://ishaan010.github.io/ezpie/python/data-type/numbers.html) please remake it. But I really liked it

**Comment by ishaan010** [comment]
And also move the file to the data type folder where all the webpages of data type are

File: python/basic/intro.html [hunk 1]
```diff
@@ -47,6 +47,7 @@ <h4 class="head">Python basic</h4>
                     <li><a class="learn" href="inOut.html">Python I/O</a></li>
                     <li><a class="learn" href="operators.html">Operators</a></li>
                     <li><a class="learn" href="conversion.html">type conversion</a></li>
+                    <li><a class="learn" href="lists.html">Lists</a></li>
```
Comment (ishaan010) [inline python/basic/intro.html:50]: This is not needed as all files are edited as per there folders

Comment (Beatrix-droid) [reply python/basic/intro.html:50]: Noted, will make the changes and commit by this evening

Comment (ishaan010) [reply python/basic/intro.html:50]: OK thanks for your help

Comment (ishaan010) [reply python/basic/intro.html:50]: And one more thing, when you said 'this evening' do you mean the time of your country or India?

Comment (Beatrix-droid) [reply python/basic/intro.html:50]: Ah sorry, I am living in the uk.
By, we are 5 hours behind.
I was going to get it done by midnight of today uk time (so by 3am in india)

Comment (ishaan010) [reply python/basic/intro.html:50]: OK

Comment (ishaan010) [reply python/basic/intro.html:50]: Have you made the changes and I have also added you on my project todo list if you like to keep on helping you can do it by following the todos

**Pull request closed without merging**